generator/generator_tool/generator_tool.cpp

```diff
@@ -180,8 +180,9 @@ DEFINE_string(uk_postcodes_dataset, "", "Path to dataset with UK postcodes.");
 DEFINE_string(us_postcodes_dataset, "", "Path to dataset with US postcodes.");
 
 // Printing stuff.
-DEFINE_bool(calc_statistics, false, "Calculate feature statistics for specified mwm bucket files.");
-DEFINE_bool(type_statistics, false, "Calculate statistics by type for specified mwm bucket files.");
+DEFINE_bool(calc_stats, false, "Print file and feature stats.");
```

Comment:
calc_ is not needed here and below

Reply:
a common prefix is needed to keep all 3 options together in the `--help` output

Reply:
-stats, -stats_type (or -stats-type, or better both :) ), -stats_geom